base/compiler/ssair/show.jl

```diff
@@ -104,45 +116,63 @@ function show_unquoted_phinode(io::IO, stmt::PhiNode, indent::Int, prefix::Strin
     print(io, ')')
 end
 
-function show_unquoted(io::IO, stmt::PhiCNode, indent::Int, ::Int)
+function show_unquoted(io::IO, stmt::PhiCNode, indent::Int, prec::Int, unstable_ssa::BitSet = BitSet())
     print(io, "φᶜ (")
     first = true
     for v in stmt.values
         first ? (first = false) : print(io, ", ")
-        show_unquoted(io, v, indent)
+        if v isa SSAValue
+            show_unquoted(io, v, indent, prec, (v in unstable_ssa) ? true : false)
```

Comment:
Option suggestion, but: how about changing this and its many variants in this PR to
```suggestion
            show_unquoted(io, v, indent, prec, unstable_ssa)
```
and then deal with it in the `show_unquoted(::IO, ::SSAValue, ...)` method. 

That would then also make it trivial to use `unstable_ssa::Union{BitSet,Nothing} = nothing` everywhere to avoid creating a `BitSet()` if it is not needed. Dealing with the `nothing` case would only have to be done in one place, namely the `show_unquoted` method for `SSAValue`

Reply:
I remember I had to do that to avoid stackoverflow. But I will check again. Reasonable suggestion to invest time. Thanks.